bin/mail/send-to-mailing-list

```diff
@@ -35,7 +35,7 @@ end
 def process_command_line(argv=ARGV)
   {
     csv: nil,
-    limit: 100000,
+    limit: nil,
```

Comment:
Why is the default being removed? If anything, I'd encourage lowering the default value to prevent accidental mishaps, allowing the caller to specify an appropriate limit for the particular email campaign.